mlir/utils/jenkins/Jenkinsfile

```diff
@@ -544,9 +552,11 @@ pipeline {
                         }
                         steps {
                             buildProject('ci-performance-scripts', '')
+                            // How to check out into specific directory, according to stackoverflow.
```

Comment:
Check out MITuna this way rather than as a shell command.  I've been seeing lockharts fail with apparent proxy problems, and I think this'll avoid them.  Also it's arguably cleaner.